### DISCUSSION TITLE
Suggestion: use Algolia for Search

### DISCUSSION THREAD

**Discussion by osseonews** [graphcommerce-org/graphcommerce#1383] [Ideas]
Just a suggestion. For the search, I'd use Algolia. Easy to implement for this type of framework. Just send the product data from Magento into Algolia. I believe there is an Algolia extension for Magento, but not really needed either (try to limit all extension in Magento), because you can just populate the index with some sort of Node API in nextjs.

**Comment by paales** [comment]
Yeah! We've got some ideas how to implement this.

In an ideal scenario I'd like be able to index _any_ data and send that to Algolia or something else and not be reliant on Magento to index stuff. I think we can greatly reduce indexing complexity and give way more control to developers.

Do you have any examples that show what it should look like in GraphCommerce?
